src/store/LocalStorage.tsx

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
Why are we using local storage?

Reply:
removed local storage, page refreshing was causing issues, but ended up fixing the issues, but did not remove local storage.